src/core/modules/Planet.cpp

```diff
@@ -4890,10 +4893,40 @@ void Planet::drawOrbit(const StelCore* core)
 
 	sPainter.setColor(getCurrentOrbitColor(), orbitFader.getInterstate());
 	Vec3d onscreen;
+
+	if (!fromMoonPerspective) {
+		// Update the orbit positions to the current planet date.
+		computeOrbit();
+	}
+	else {
+		double dateJDE = lastJDE;
+		double calc_date;
+
+		Vec3d myPos = core->getCurrentPlanet()->getHeliocentricEclipticPos(lastJDE);
+		Vec3d myparentPos = core->getCurrentPlanet()->getParent()->getHeliocentricEclipticPos(dateJDE);
+
+		// pretend they are on x-y plane
```

Comment:
I have changed the code to project the vector of the moon onto the orbital plane of my parent planet. Cant simply use dot product because we need the sign of the angle too whether the angle is in front or behind the parent planet current location or some moon orbital plane is perpendicular to the parent planet orbital plane.